app/flatpak-builtins-config.c

```diff
@@ -227,12 +247,32 @@ get_config_key (const char *arg, GError **error)
   return NULL;
 }
 
+static FlatpakDir *
+get_config_dir(FlatpakDir *dir, ConfigKey *key)
+{
+  switch (key->scope)
+    {
+      case CONFIG_SCOPE_INSTALLATION:
+        return dir;
+      case CONFIG_SCOPE_USER:
+        return flatpak_dir_get_user ();
+    }
+
+    g_printerr ("invalid config scope");
+    return NULL;
+}
+
 static char *
 print_config (FlatpakDir *dir, ConfigKey *key)
 {
   g_autofree char *value = NULL;
+  FlatpakDir *configDir = NULL;
```

Comment:
```suggestion
  FlatpakDir *config_dir = NULL;
```

This is the proper casing for a variable.